backend/src/agents/__init__.py

```
@@ -232,16 +241,56 @@ def init_config(
+    if DaytonaSandbox is None or Daytona is None:
+        return None, None
+
+    key = DAYTONA_API_KEY
+    if not key:
+        return None, None
+
+    try:
+        client = Daytona(DaytonaConfig(api_key=key))  # type: ignore[misc]
+        sandbox = client.create()
+        backend = DaytonaSandbox(sandbox=sandbox)
+        return sandbox, backend
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)